src/stores/spaces.ts

```diff
@@ -1,6 +1,8 @@
 import { defineStore } from 'pinia';
 import { currentNetwork } from '@/networks';
+import { useStorage } from '@vueuse/core';
 import type { Space } from '@/types';
+import pkg from '@/../package.json';
```

Comment:
```suggestion
import pkg from '../../package.json';
```

Going back an alias looks a bit ugly, let's use relative import for now.